src/main/resources/migration/sql/001_Basic_tables.sql

```diff
@@ -0,0 +1,68 @@
+CREATE SCHEMA messaging;
```

Comment:
To make it simpler for reading, could we please add more newlines between different entities' definition?

For example, it might look like this:
```sql
CREATE SCHEMA messaging;

-- Person

CREATE TABLE IF NOT EXISTS messaging.person(
    person_id uuid NOT NULL,
    name character varying(50) NOT NULL,
    date_of_birth timestamp with time zone,
    person_info character varying(300),
    CONSTRAINT person_table_pk PRIMARY KEY (person_id)
);

COMMENT ON TABLE messaging.person IS 'Данные пользователя';
COMMENT ON COLUMN messaging.person.person_info IS 'Информация в вольном стиле от пользователя о себе';


-- Dialog

CREATE TABLE IF NOT EXISTS messaging.dialog (
...
```

Reply:
Yep, I'll correct this